interpreter/src/main/scala/pl/touk/nussknacker/engine/compile/nodecompilation/GenericNodeTransformationValidator.scala

```diff
@@ -181,39 +181,26 @@ class GenericNodeTransformationValidator(
           )
           .sequence
         params
-          .andThen { branchParams =>
-            branchParams
-              .map { case (branchId, expression) =>
-                Validations.validate(
-                  parameter,
-                  evaluatedparam.Parameter(ParameterNaming.getNameForBranchParameter(parameter, branchId), expression)
```

Comment:
What happened with this code?

Reply:
Validation is now done after compilation :)

Reply:
Ok, but what about `getNameForBranchParameter`? Thanks to that, errors are shown in the exact place in the form. Branch parameters have two levels: one is parameter level and next one is expression for the given edge. You need to identify this two level's path to recognize where is the error.

Reply:
Nice, thx